src/psyclone/tests/transformations_test.py

```diff
@@ -236,104 +236,38 @@ def test_regiontrans_wrong_options():
 # Tests for ProfileRegionTrans
 
 
-def dummy_nodes():
-    '''Utility routine which create a minimal invoke and schedule
-    hierarchy for use in subsequent ProfileRegionTrans tests.
-
-    :returns: a list containing a single node connected to a minimal \
-    InvokeSchedule which is connected to a minimal Schedule. This \
-    hierarchy is required for code unrelated to the tests.
-    :rtype: list of :py:class:`psyclone.psyGen.Node`
-
-    '''
-    schedule = InvokeSchedule(None, None)
-    schedule.invoke = Invoke(None, None, None)
-    node = Node()
-    node.parent = schedule
-    nodes = [node]
-    schedule.children = nodes
-    return nodes
-
-
-def test_profilerregiontrans_noname(monkeypatch):
-    '''Check that when no name is supplied to the profile transformation
-    then it creates a profile node with no name information.
+@pytest.mark.parametrize("options", [None, {"invalid": "invalid"},
+                                     {"profile_name": ("mod", "reg")}])
+def test_profileregiontrans_name(options):
+    '''Check that providing no option or an option not associated with the
+    profile transformation does not result in anything being passed
+    into ProfileNode via the name argument and that providing an
+    option associated with the profile transformation does result in
+    the the relevant names being passed into ProfileNode via the name
```

Comment:
I don't think we want names relevant to "The The" :-)

Reply:
:-)